lib/rsa/rsa-verify.c

```diff
@@ -321,11 +321,12 @@ static int rsa_verify_key(struct image_sign_info *info,
 #if !defined(USE_HOSTCC)
 	ret = uclass_get_device(UCLASS_MOD_EXP, 0, &mod_exp_dev);
 	if (ret) {
-		printf("RSA: Can't find Modular Exp implementation\n");
-		return -EINVAL;
+		printf("RSA: No Modular Exp HW implementation, use SW\n");
```

Comment:
This code should probably be straightened out upstream.  The check for `USE_HOSTCC` is very odd.  However, we shouldn't hit a problem in this code as it is, because we are using `SPL_DM`.